tremor-script/src/srs.rs

```diff
@@ -248,6 +551,79 @@ impl Stmt {
 =========================================================================
 */
 
+/// A query declaration
+#[derive(Clone, PartialEq)]
+pub struct PipelineDecl {
+    /// The identity of this pipeline
+    pub id: String,
+    raw: Vec<Arc<Pin<Vec<u8>>>>,
```

Comment:
do we really need this here, nothing in the rest of the struct seems to be referencing it